dev-haskell/megaparsec-tests/megaparsec-tests-8.0.0-r1.ebuild

```diff
@@ -22,7 +22,7 @@ RDEPEND=">=dev-haskell/hspec-2.0:=[profile?] <dev-haskell/hspec-3.0:=[profile?]
 	>=dev-haskell/hspec-megaparsec-2.0:=[profile?] <dev-haskell/hspec-megaparsec-3.0:=[profile?]
 	~dev-haskell/megaparsec-8.0.0:=[profile?]
 	>=dev-haskell/mtl-2.2.2:=[profile?] <dev-haskell/mtl-3.0:=[profile?]
-	>=dev-haskell/quickcheck-2.10:2=[profile?] <dev-haskell/quickcheck-2.15:2=[profile?]
```

Comment:
So, this dep seems to have  been bumped in [rev1 upstream][1] (even though they don't mention it in the change log)

I've been trying to pull these upstream revisions where possible, you can see an example [in this ebuild][2], and a little bit of conversation [here in my PR][3].

Perhaps I should open an issue to discuss it, as it seems the conversation is lost/buried in the PR....

Either way, for the PR itself I'd like to keep this consistent, so can you update your ebuild to do the same?

[1]: https://hackage.haskell.org/package/megaparsec-tests-9.0.1/revision/1.cabal
[2]: https://github.com/ezzieyguywuf/gentoo-haskell/blob/be1f3268448fe8bff172a4660210512c809ba4fb/dev-haskell/kansas-comet/kansas-comet-0.4-r6.ebuild#L15
[3]: https://github.com/gentoo-haskell/gentoo-haskell/pull/1118#discussion_r556518075